challenges/functions.js

```diff
@@ -1,32 +1,54 @@
-// ==== Callbacks ====  
+// ==== Callbacks ====
 
 /* Step 1: Create a higher-order function that accepts a callback
-  * Create a higher-order function named consume that can take 3 parameters.
-  * The first two parameters can accept any argument
-  * The last parameter accepts a callback 
-  * In the body of the function return the callback with the two parameters that you created
-*/
+ * Create a higher-order function named consume that can take 3 parameters.
+ * The first two parameters can accept any argument
+ * The last parameter accepts a callback
+ * In the body of the function return the callback with the two parameters that you created
+ */
 
+const consume = (first, second, last) => {
+  return last(first, second);
+};
+
+const lastParam = (a, b) => {
+  return `${a} ${b}`;
+};
+console.log(consume("Hello", ":)", lastParam)); // Hello :)
 
 /* Step 2: Create several functions to callback with consume();
-  * Create a function named add that returns the sum of two numbers
-  * Create a function named multiply that returns the product of two numbers 
-  * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
-*/
+ * Create a function named add that returns the sum of two numbers
+ * Create a function named multiply that returns the product of two numbers
+ * Create a function named greeting that accepts a first and last name and returns "Hello first-name last-name, nice to meet you!"
+ */
+
+const add = (a, b) => {
+  return a + b;
+};
+console.log(consume(10, 5, add)); //15
 
+const multiply = (a, b) => {
+  return a * b;
+};
+console.log(consume(10, 5, multiply)); //50
+
+const greeting = (a, b) => {
```

Comment:
Maybe update the naming here. It works, but makes more sense if we do `(firstName, lastName) =>` etc. Other than that, the ES6 all looks great.